### PR TITLE
ci: add CD and semantic-release

### PR DESCRIPTION
**Pull request by Rei-x**

### CODE REVIEW

File: .releaserc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "@semantic-release/github",
+    "@semantic-release/npm",
+    [
+      "@semantic-release/git",
+      {
+        "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
+      }
+    ]
+  ]
+}
```
Comment (Byczax) [inline .releaserc:30]: What this file do?

**Review by Byczax — APPROVED** [review]
Good for me!

File: .releaserc (hunk 1 above)
Comment (Rei-x) [reply .releaserc:30]: its config file for semantic release, so it will automatically generate release notes for new deployments

**Pull request merged**